modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java

```diff
@@ -75,7 +75,8 @@
 
 		EditorConfiguration editorConfiguration =
 			EditorConfigurationFactoryUtil.getEditorConfiguration(
-				StringPool.BLANK, ddmFormFieldType, "ckeditor_classic",
+				JournalPortletKeys.JOURNAL, ddmFormFieldType,
```

Comment:
Of course @fortunatomaldonado,

On line 78 you are passing `JournalPortletKeys.JOURNAL` as an argument.
Instead of doing this way, you could extract the `portletName` from the `portletNamespace`, and pass it as an argument on `JournalPortletKeys.JOURNAL` place.

Why doing this way is better?
Because like this we have a chance to pass the correct `portletName` everytime, independent of the portlet which is calling the method `EditorConfigurationFactoryUtil.getEditorConfiguration(...`.

Hereupon, the method call should be like this:
```
EditorConfigurationFactoryUtil.getEditorConfiguration(
				portletName, ddmFormFieldType,...
```

Where `porletName` will be a string extracted from the `portletNamespace`. For example:

```
String portletNamespace  = ddmFormFieldRenderingContext.getPortletNamespace();
String portletName = portletNamespace.substring(1, portletNamespace.length()-1); 
```


In the end, it will turn in something like this: (starting from line 76)

```
String portletNamespace =
	ddmFormFieldRenderingContext.getPortletNamespace();
String portletName =
	portletNamespace.substring(1, portletNamespace.length() - 1);

EditorConfiguration editorConfiguration =
	EditorConfigurationFactoryUtil.getEditorConfiguration(
		portletName, ddmFormFieldType, "ckeditor_classic",...
```

Reply:
Truly appreciate the thorough explanation! I will be sending this out shortly.